src/node/history.h

```diff
@@ -607,9 +607,9 @@ namespace ccf
       size_t all_data_size) override
     {
       append(replicated, replicated_size, all_data, all_data_size);
+#ifdef PBFT
```

Comment:
You can use `store.get_consensus()` here to determine if we are running with pbft or raft (look at `is_replicated_tree_enabled()` ) instead of using the compile time flag